bindata/etcd/cluster-restore.sh

```diff
@@ -67,6 +67,15 @@ function wait_for_containers_to_stop() {
   done
 }
 
+function check_snapshot_status() {
```

Comment:
would it make sense for this to be a shared function?

Reply:
In future, we may be extending the function to include generation of the hash. The function provides a logical unit for a set of instructions and make the flow easy to follow and maintain.

Reply:
Thanks. Well, earlier we were doing different things in `backup` and `restore` and hence didn't think about making it a shared function. Now that the functionality is identical, I made it a common function and moved it to common-tools.